pkg/instrumentation/auto/annotation_test.go

```diff
@@ -106,6 +106,7 @@ func TestAnnotationMutators_Namespaces(t *testing.T) {
 			ctx := context.Background()
 			client := fake.NewClientBuilder().WithLists(&corev1.NamespaceList{Items: namespaces}).Build()
 			mutators := NewAnnotationMutators(
+				client,
```

Comment:
did you mean clientReader?

Reply:
The client reader is just an interface. We can reuse the fake client for both.